src/submission/submission-service.ts

```diff
@@ -845,8 +845,17 @@ export namespace operations {
     schema: dictionaryEntities.SchemasDictionary,
   ) => {
     const schemaName = command.clinicalType as ClinicalEntitySchemaNames;
+
     // check records are unique
-    const errors: SubmissionValidationError[] = checkUniqueRecords(schemaName, command.records);
+    let useAllRecordValues = false;
+    if (schemaName === ClinicalEntitySchemaNames.BIOMARKER) {
+      useAllRecordValues = true;
```

Comment:
The main point of this pr is to check if the entire record is unique, because biomarker has 5 unique id columns.

Reply:
can you add a comment above this with the link to the ticket of the bug

Reply:
done!